templates/home/_fsf_ruby.html

```diff
@@ -22,6 +23,7 @@ <h4>Here’s how mdl uses it:</h4>
 
 <b>parts</b>:
   <b>mdl</b>:
+    <b>source:</b>: .
     <b>plugin</b>: ruby
     <b>gems</b>:
       - mdl
```

Comment:
This will install mdl from rubygems.org, not source (`source: .`). We can go down that road, but we should be explicit that this isn't building from source but rather fetching the gem from rubygems.org and bundling it into a snap.

Alternatively, the install instructions for mdl suggest building with rake. This produced a working snap from source for me:
```
name: mdl
version: "0.5.0"
summary: Markdown lint tool
description: |
  Style checker/lint tool for markdown files

confinement: devmode
base: core18

parts:
  mdl:
    source: .
    plugin: ruby
    gems:
      - rake
      - bundler
    override-build: |
      snapcraftctl build
      rake install
    stage-packages:
      - git

apps:
  mdl:
    command: mdl
```

Reply:
I like the second option. I will update the files and description accordingly.